src/handlers/comment/handlers/wallet.ts

```diff
@@ -47,6 +48,26 @@ export const registerWallet = async (body: string) => {
     logger.info(`Resolved address from Ens name: ${ensName}, address: ${address}`);
   }
 
+  const regexForSigHash = /(0x[a-fA-F0-9]{130})/g;
+  const sigHashMatches = body.match(regexForSigHash);
+  const sigHash = sigHashMatches ? sigHashMatches[0] : null;
+
+  const messageToSign = "DevPool";
+  const failedSigLogMsg = `Skipping to register the wallet address because you have not provided a valid SIGNATURE_HASH, SIGNATURE_HASH: ${sigHash}`;
```

Comment:
I think it's unnecessary to output the invalid signature hash here because it will be visible in the previous comment just above.